test/integration/image-component/default/pages/valid-html-w3c.js

```diff
@@ -0,0 +1,20 @@
+import Head from 'next/head'
+import Image from 'next/image'
+
+const Page = () => {
+  return (
+    <div>
+      <Head>
+        <title>Title</title>
+        <meta name="description" content="Description..." />
+        <link rel="icon" type="image/jpeg" href="/test.jpg" />
+      </Head>
```

Comment:
Maybe this should be in the `_document.js` and then we can run validation against every page?